node/pkg/watchers/ibc/wasm_attrs.go

```diff
@@ -0,0 +1,102 @@
+package ibc
+
+import (
+	"encoding/base64"
+	"fmt"
+	"strconv"
+
+	"github.com/tidwall/gjson"
+
+	"go.uber.org/zap"
+)
+
+// WasmAttributes is an object to facilitate parsing of wasm event attributes. It provides a method to parse the attribute array in a wasm event,
+// plus methods to return attributes as the appropriate type, including doing range checking.
+type WasmAttributes struct {
+	m map[string]string
+}
+
+// GetAsBytes returns the attribute value as a string.
```

Comment:
Did you mean to say generic "GetAsBytes" or does this need to be "GetAsString"?

Reply:
Fixed the copy/paste errors.